docs/configuration/replication.md

```diff
@@ -0,0 +1,11 @@
+**Things to avoid**
+
+1. Don't replicate an image without specifying the tag you want. Specifically,
+   through the harbor UI as this will mean you pull all versions of the images. If you cancel this process it may cause errors when you try to pull in the future.
+
+   Because of this you should only replicate images that you want a specfic tag from. We would need to look more into this bug to determine exactly what is happening.
```

Comment:
Is it possible to set a version constrain instead of fixed versions, like saying we want anything that is 3.x, or <=7.2.0?

Reply:
Also are there some upstream tickets you know of that are related to this issue? Would you link them please?